culture/social-rules.md

```diff
@@ -0,0 +1,82 @@
+```
+
+A well-actually is when you correct someone about something that’s not relevant to the conversation or tangential to what they’re trying to say.[^1] They’re bad because they aren’t helpful, break the flow of conversation, and focus attention on the person making the well actually.
+
+This rule can be a bit tricky because there isn’t a clear line between relevant to the conversation and not. Sometimes your correction might actually be necessary, and it could still come off as annoying when you make it. The best rule of thumb is, if you’re not sure whether something needs to be said right now, hold off and see what happens. You can always say it later if it turns out there’s no way for the conversation to move forward without your correction.
+
+## No feigning surprise
+
+```{epigraph}
+> **Dan**: What’s the command line?
+> 
+> **Carol**: Wait, you’ve never used the command line?
+```
+
+Feigned surprise is when you act surprised when someone doesn’t know something. Responding with surprise in this situation makes people feel bad for not knowing things and less likely to ask questions in the future, which makes it harder for them to learn.
+
+_No feigning surprise_ isn’t a great name. When someone acts surprised when you don’t know something, it doesn’t matter whether they’re pretending to be surprised or actually surprised. The effect is the same: the next time you have a question, you’re more likely to keep your mouth shut. An accurate name for this rule would be _no acting surprised when someone doesn’t know something_, but it’s a mouthful, and at this point, the current name has stuck.
+
+
+## No backseat driving
+
+```{epigraph}
+> **Bob**: What’s the best tool for data science with tables?
+> 
+> **Alice**: NumPy.
+> 
+> **Eve**: _(from across the room)_ No, you should use Pandas. It’s better.
+```
+
+Backseat driving is when you lob advice from across the room (or across the online chat) without really joining or engaging in a conversation. Because you haven’t been participating in the conversation, it’s easy to miss something important and give advice that’s not actually helpful. Even if your advice is correct, it’s rude to bust into a conversation without asking. If you overhear a conversation where you could be helpful, the best thing to do is to ask to join.
+
+(social:subtle-isms)=
+## No subtle -isms
+
+```{epigraph}
+> **Carol**: Windows is hard to use.
+> 
+> **Bob**: No way. Windows is so easy to use that even my mom can use it.
+```
+
+Subtle -isms are subtle expressions of racism, sexism, ageism, homophobia, transphobia and other kinds of bias and prejudice. They are small things that make others feel unwelcome, things that we all sometimes do by mistake. Subtle -isms make people feel like they don’t belong in the 2i2c community. We want to create an environment where everyone can focus all their energy on interactive computing. It’s hard to do that if you’re regularly being made to wonder whether you belong.
```

Comment:
This overlaps with the CoC. I don't think it should appear in both the "unwelcoming behavior" category and the "social rules". The former is significantly more serious.